boards/sipeed-longan-nano/include/board.h

```diff
@@ -53,6 +53,27 @@ extern "C" {
 #define LED_BLUE_PIN        LED2_PIN    /**< LED2 is blue */
 /** @} */
 
+#if defined(MODULE_SDCARD_SPI)
+#define SDCARD_SPI_PARAM_SPI         SPI_DEV(0)
+#define SDCARD_SPI_PARAM_CS          GPIO_PIN(PORT_B, 12)
+#define SDCARD_SPI_PARAM_CLK         GPIO_PIN(PORT_B, 13)
+#define SDCARD_SPI_PARAM_MISO        GPIO_PIN(PORT_B, 14)
+#define SDCARD_SPI_PARAM_MOSI        GPIO_PIN(PORT_B, 15)
+#endif
```

Comment:
You probably also want to provide MTD config if you add this (reminds me that we should add a generic MTD config for SD cards, this is currently just copy & paste across all boards)

Reply:
> https://github.com/RIOT-OS/RIOT/pull/19216 should automatically do this for you

:+1:

Reply:
So it makes sense to wait for your PR.

Reply:
> I could as well add the MTD device in my PR if SPI support gets merged first.

Ok.